framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/WorkbenchWithKieServerScenarioImpl.java

```diff
@@ -72,8 +72,6 @@ public WorkbenchWithKieServerScenarioImpl(OpenShiftController openShiftControlle
         envVariables.put(OpenShiftTemplateConstants.IMAGE_STREAM_NAMESPACE, projectName);
         project.processTemplateAndCreateResources(OpenShiftConstants.getKieAppTemplate(), envVariables);
 
-        waitUntilAllPodsAreReady(project);
```

Comment:
Please also delete private method waitUntilAllPodsAreReady(Project project) as this is the only place where it is used.